tests/helpers/generate-table.js

```diff
@@ -0,0 +1,135 @@
+import hbs from 'htmlbars-inline-precompile';
+import { A as emberA } from '@ember/array';
+import wait from 'ember-test-helpers/wait';
+
+const alpha = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+function hex(a) {
+  // First figure out how many digits there are.
+  a += 1; // This line is funky
+  let c = 0;
+  let x = 1;
+  while (a >= x) {
+    c++;
+    a -= x;
+    x *= 26;
+  }
+
+  // Now you can do normal base conversion.
+  let s = '';
+  for (let i = 0; i < c; i++) {
+    s = alpha.charAt(a % 26) + s;
+    a = Math.floor(a / 26);
+  }
+
+  return s;
+}
+
+const fullTable = hbs`
```

Comment:
There are many duplication of this helper file and the ones in `test-scenario`. Can you merge them?

Reply:
That’s the plan! The APIs are different so I didn’t want to block this PR on them, the next one will be cleanup and expand tests in general, then Tree model refactors

Reply:
Sounds good to me